### ISSUE TITLE
Issues with _util.py on Linux

### ISSUE THREAD

**Issue by kerstin** [isislovecruft/python-gnupg#145]
Heya, line 378 of _util.py gives me an invalid syntax error on Linux on import of gnupg...

```
    assert line.upper().startswith(u''.join(prefix).upper())
                                     ^
SyntaxError: invalid syntax

**Comment by isislovecruft** [comment]
What version of python are you using?

**Comment by CrystallineCat** [comment]
The `u` prefix for string literals was removed from the syntax in Python 3.0, then added back in as a no-op in 3.3 (see https://www.python.org/dev/peps/pep-0414/).

To support all versions, you can use the respective text type constructor (`str` or `unicode`) without arguments, which will create an empty string:

    (str if _py3k else unicode)().join(...)

Alternatively you can use the `six` module from pip to get the correct type:

    six.text_type().join(...)

**Comment by sigmavirus24** [comment]
@CrystallineCat correct. On top of that the versions of Python 3 which don't support `u` prefixing should have already reached their End of Life. So, I don't see anything wrong with using `u` prefixed strings here.
